### PR TITLE
Add zeromq405 formula

### PR DESCRIPTION
**Pull request by cvortmann**
- Checked out version `8dd3478` from homebrew/homebrew.
- Removed bottle block.
- Changed SHA1 to SHA256 in stable block.
- Fix code style reported with `brew audit --strict`.

### CODE REVIEW

**Comment by cvortmann** [comment]
That checks failed to build for `el capitan`. How should I fix that?

**Comment by cvortmann** [comment]
Should I remove the `head`block? What about `stable/patch`?

File: zeromq405.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+class Zeromq405 < Formula
+  desc "High-performance, asynchronous messaging library"
+  homepage "http://www.zeromq.org/"
+  url "http://download.zeromq.org/zeromq-4.0.5.tar.gz"
+  sha256 "3bc93c5f67370341428364ce007d448f4bb58a0eaabd0a60697d8086bc43342b"
+  revision 2
```
Comment (DomT4) [inline zeromq405.rb]: Don't need `revision` line on a new formula.

File: zeromq405.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  conflicts_with "zeromq", :because => "Differing version of the same formula"
+
+  stable do
+    patch do
```
Comment (DomT4) [inline zeromq405.rb]: You can put this in the main formula now & kill the `head` block. Versions formulae don't generally have `head`s.

File: zeromq405.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      # enable --without-libsodium on libzmq < 4.1
+      # zeromq/zeromq4-x#105
+      url "https://gist.githubusercontent.com/minrk/478aab66adf7016158ff/raw/b5ea2d61c3f66db6ff3e266b76d1bec4ad4a238b/without-libsodium.patch"
+      sha1 "68543ff1b0f64b22994cb13b4d24bce8f76cf431"
```
Comment (DomT4) [inline zeromq405.rb]: Use `sha256` for this one please.

File: zeromq405.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  option :universal
+  option "with-libpgm", "Build with PGM extension"
+
+  deprecated_option "with-pgm" => "with-libpgm"
```
Comment (DomT4) [inline zeromq405.rb]: No such thing as a `deprecated_option` for a new formula :)

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/5b338e9d551380e2e37fcc80e1126cf90a9cf01b. Thank you for your contribution to Homebrew @cvortmann! :smiley_cat:

**Comment by cvortmann** [comment]
YEY!

**Pull request closed without merging**